src/com/itschool/session4/homework/challenge03/LogicalOperators.java

```diff
@@ -0,0 +1,25 @@
+package com.itschool.session4.homework.challenge03;
+
+import java.util.Scanner;
+public class LogicalOperators {
+
+    public static void main(String[] args) {
+        //creaza un nou obiect  Scanner
+        Scanner obiecScaner = new Scanner(System.in);
+
+
+        // Prompt pentru introducere numar
+        System.out.println("Introduceti primul numar");
+        int numar1 = obiecScaner.nextInt();
+
+        boolean isPositive = !(numar1<0);
+
+        if(isPositive){
+            System.out.println("Numarul este pozitiv");
+        } else{
```

Comment:
nice to see you used **if-else** statements even though we haven't studied them yet. However, you could have solved this also with stuff studied so far. Something like (one liner):
`System.out.println("Is the number positive? " + isPositive);`
Also, I see a mixing of words/variables in both Romanian and English. You should stick to one only (preferably English).

Reply:
ok. I am also more comfortable with English than RO in IT.